CHANGELOG.md

```diff
@@ -12,6 +12,11 @@
 * Adjusted code to not trigger warnings with PHP 8.4
 * Removed deprecated `getDeserializeFormat` method from date time
 * Replaced `@Preferred` annotation with the `#[Preferred]` attribute
+* Added a new `PropertyNamingStrategyInterface`  which can be used to modify the naming strategy for properties. Previously 
```

Comment:
```suggestion
* Replaced `PropertyCollection::useIdenticalNamingStrategy` with the `PropertyNamingStrategyInterface`. To change the default strategy, you can now inject it into the parser instead of changing a static property.
```